lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -265,10 +284,13 @@ private JourneyResponse getJourneyResponse(
                     Optional.ofNullable(clientOAuthSessionItem.getReproveIdentity());
 
             if (reproveIdentity.orElse(false) || configService.enabled(RESET_IDENTITY)) {
-                LOGGER.info(
-                        LogHelper.buildLogMessage(
-                                "resetIdentity flag is enabled, reset users identity."));
-                return JOURNEY_REPROVE_IDENTITY;
+                if (preferredNewIdentityLevel == Vot.P1) {
```

Comment:
This section has changed slightly since the original PR so that we have two different log messages depending on the vtr